src/test/resources/features/DeleteUserRole.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Delete UserRole
+  In order to interact with a group
+  As a user
+  I want to leave a group
+
+  Scenario: Leave a group being subscribed
+    Given There is a registered user with username "user" and password "password" and email "user@sample.app"
+    And I login as "user" with password "password"
+    And A group exists
+    And The user "user" belongs to that group as "SUBSCRIBED"
+    When The user "user" leaves the group
+    Then The response code is 204
+
+  Scenario: Leave a group being admin and there are other admins in the group
```

Comment:
This and the next scenario descriptions seem interchanges. For this one, the given situation is that there is just one admin, though the description mentions there are others. The reverse for the following one.